### PR TITLE
Moved repository from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by evaherrada**

### CODE REVIEW

**Comment by evaherrada** [comment]
Personally, I think the best course of action for this would be reverting the most recent commit

**Comment by siddacious** [comment]
@dherrada This is failing for a documentation build issue and it was already failing, though it may have been hard to tell because what looks like a typo in the cookiecutter setup broke the link to Travis. It's here:
https://travis-ci.com/adafruit/Adafruit_CircuitPython_CPython

Can you give more detail about why you were suggesting reverting the most recent commit?

**Comment by evaherrada** [comment]
@siddacious Prior to the most recent commit, the only issues were pylint issues in the docstring of adafruit__cpython.py. In fixing the pylint issues, I messed up some of the rst which caused it to fail.

File: adafruit__cpython.py [hunk 1]
```diff
@@ -32,15 +32,17 @@
 
 **Hardware:**
 
-.. todo:: Add links to any specific hardware product page(s), or category page(s). Use unordered list & hyperlink rST
+.. todo:: Add links to any specific hardware product page(s), or category
```
Comment (siddacious) [inline adafruit__cpython.py]: Your attempts to get this passing are admirable but RST doesn't play fair. You can undo this.

File: adafruit__cpython.py [hunk 2]
```diff
@@ -32,15 +32,17 @@
...
    inline format: "* `Link Text <url>`_"
 
 **Software and Dependencies:**
 
 * Adafruit CircuitPython firmware for the supported boards:
   https://github.com/adafruit/circuitpython/releases
-  
-.. todo:: Uncomment or remove the Bus Device and/or the Register library dependencies based on the library's use of either.
+
+.. todo:: Uncomment or remove the Bus Device and/or the Register library
```
Comment (siddacious) [inline adafruit__cpython.py]: This can be reverted as well

**Review by siddacious — CHANGES_REQUESTED** [review]

**Comment by siddacious** [comment]
@dherrada ah! I see what you mean. Ya, go ahead and revert the TODO changes

**Comment by evaherrada** [comment]
@siddacious Ok. Will rebase in 1 sec

**Review by siddacious — APPROVED** [review]

**Pull request merged**